content/posts/xapi-activities-101/index.md

```diff
@@ -0,0 +1,262 @@
+
+### The Experience API
+
+We suppose the reader is familiar with the existence of the
+[Experience API (xAPI)](https://github.com/adlnet/xAPI-Spec) and to some extent to the concept of
+[xAPI statement](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-About.md#def-statement).
+
+> For french readers take a look at the
+> [blog of our friend Sébastien Fraysse](https://www.learningdata.fr/xapi-en-bref/) who did a
+> terrific job at making the learning data world understandable and .. enchanting!
+
+The Experience API (xAPI) is the international standard for learning data.  
+In their own words it's _a technical specification that aims to facilitate the documentation and
+communication of learning experiences._
+
+It is open source and can be found on [GitHub](https://github.com/adlnet/xAPI-Spec).
+
+### What we'll talk about
+
+We're going to discuss the concepts behind the xAPI representation of activities and what it means
+to
+[Learning Record Producers](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-About.md#def-learning-record-provider)
+(LRP, like [marsha](https://github.com/openfun/marsha)),
+[Learning Record Stores](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-About.md#def-learning-record-store)
+(LRS, like [ralph](https://github.com/openfun/ralph)) and
+[Learning Record Consumers](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-About.md#def-learning-record-consumer)
+(LRC, like [warren](https://github.com/openfun/warren)) We'll try to make the concept of activity
+clearer for all parties so that anyone can make enlightened decisions about how to share or query
+activity definitions and metadata!
+
+### TL;DR
+
+The term `activity` appears in different sections of the specification and the different concepts
+introduced by the spec are somewhat intertwined with one another. To make the following explanations
+easier to grasp, we'll present them step by step according to the masterful plan below :
+
+- What is an activity according to the spec?
+- How is an activity modeled? That is, what is its structure?
+- Where can we find an activity's definition or additional metadata?
+- How do the different activity definitions source interact with each other to produce a correct
+  activity representation?
+
+## The xAPI activity ⚽
+
+Let's start with a philosophical question: what exactly is life? ... Sorry I meant, what exactly is
+an activity?
+
+According to the spec, an activity is
+[_a type of Object making up the "this" in "I did this"_](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-About.md#def-activity).
+_In the Statement "Anna tried a cake recipe 🎂", the recipe constitutes the Activity in terms of the
+xAPI Statement._
+
+From a learning record producer (LRP) point of view, activities will often solely exist in xAPI
+statements ; statements being the only way to transmit activities to a Learning Record Store (LRS).
+We'll see that from an LRS or LRC point of view, things can get more involved.
```

Comment:
_involved_? Do you mean "complex"?

Reply:
yes ! thought it would sound more elegant